.github/workflows/ingestion-service.yml

```diff
@@ -23,30 +30,53 @@ jobs:
     runs-on: hetzner-pm
     container:
       image: xaynetci/yellow:v11
-    timeout-minutes: 10
+    timeout-minutes: 30
     steps:
       - name: Checkout repository
         uses: actions/checkout@24cb9080177205b6e8c946b17badbe402adc938f # v3.4.0
 
       - name: Setup job
         uses: ./.github/actions/setup-job-docker
 
-      - name: Build artifacts
-        run: just build-ingestion-service ${{ github.event.inputs.features }}
+      - name: Build default artifacts
+        if: ${{ github.event.inputs.target == 'default' }}
+        run: just build-service ingestion "${{ github.event.inputs.features }}"
```

Comment:
:question: Why are we interested in keeping the x86 image?

Reply:
i will remove the x86 image